common/usb_cdc.c

```diff
@@ -157,7 +169,9 @@ static const char StrDescLanguageCodes[] = {
   0x03,			// Type is string
   0x09, 0x04	// supported language Code 0 = 0x0409 (English)
 };
-	
+
+// Note: ModemManager (Linux) ignores Proxmark3 devices by matching the
+// manufacturer string "proxmark.org". Don't change this.

```

Comment:
or if you use the 77-mm-usb-device-blacklist.rules file ,  it look for VID && PID values.

Reply:
I [patched it in ModemManager back in 2016-02](https://lists.freedesktop.org/archives/modemmanager-devel/2016-February/002705.html), prior to #287 which set a USB device ID that was registered with USB-IF.

If you have a new enough Linux distro, this works out of the box, no changes required.

Reply:
Cool.  Good to know.  Does this apply to the 'add user to dialout group' recommendation aswell?

Reply:
Nope, you still need that one.

It would be great if the vendor ID was Openmoko's, because that's been blacklisted in ModemManager since 2013.  Ubuntu's LTS channel still doesn't have the patch, because sadly a newer version of ModemManager didn't make the cut (and they don't backport rules, grumble grumble).